build.gradle

```diff
@@ -130,6 +130,16 @@ tasks.withType(FindBugs) {
   }
 }
 
+jacocoTestReport {
```

Comment:
The behaviour by default is to generate a compact report file, and call the `jacocoTestReport` task when you want a fancy HTML report. This also saves a tiny amount of build time.